### ISSUE TITLE
Synology: Does the suggested setup store the borg cache in /root?

### ISSUE THREAD

**Issue by techdaddies-kevin** [borgbase/docs.borgbase.com#11]
We had borgmatic set up on one of our Synologies, but I can't remember which method we set it up with. Long story short, Synology only provides VERY little space on /. Borg was running fine for months, but its cache was being written to /root/.borg (Or something like that) and so, a time came when / had no space available, so Synology stopped functioning correctly. We had to SSH in to figure out what was going on and to remove the cache. I haven't had a chance to reinvestigate yet, but if the setup instructions are indeed allowing Borg to write to /, they should be adjusted to cause Borg to write its cache to the shared folder location.

**Comment by m3nu** [comment]
This is true. I usually symlink the `.cache` folder to the main data folder due to that. If you have many files, this is a must.

Wanna do a PR to document this?

**Comment by techdaddies-kevin** [comment]
Yeah I can totally write up something later this week. Just to clarify - You only do this symlinking on Synology devices, right? On a standard linux machine, this isn't necessary, right?

**Comment by m3nu** [comment]
Yes, only on Syno because if you run the backup as `root` user, there is limited space in `/root`. So I link it to `/volumexx/homes/root` or something instead. It could be that `/volume/homes/root` gets copied during updates, but I'm not sure.

**Comment by markjorritsma** [comment]
I have run into this issue when setting up a 5TB backup.
The root partition of my DS415+ is (only) 2.3 GB, with approx. 86% used before setting up.

Borg quickly filled up the remaining 14% with cache files, after which it stopped.
Fortunately I did not impair the rest of my NAS' functionality much, so I could remediate the issue.

I think the documentation for Synology should be updated with a section specifically for large backups to setup a cache dir on the disks (/volumeX) instead of /.

Using this discussion I was able to setup a symlink from the /root/.cache directory to /volume1/<new cache dir>
Furthermore I excluded the /volume1/<new cache dir> in my borgmatic configuration so that it will not be included in the backup.

**Comment by m3nu** [comment]
Valid point. Maybe this can be set inside the Borgmatic config to have it all in one place.
